client_cli/Cargo.toml

```diff
@@ -25,6 +25,7 @@ maintenance = { status = "actively-developed" }
 [dependencies]
 iroha_client = { workspace = true }
 iroha_primitives = { workspace = true }
+iroha_config = { workspace = true }
```

Comment:
Shouldn't it be visible from `iroha_client`?

Reply:
`iroha_client` doesn't re-export `iroha_config_base` containing necessary traits.

Will change this dependency to `iroha_config_base` instead.